flink-connectors/flink-connector-pulsar/archunit-violations/stored.rules

```diff
@@ -0,0 +1,4 @@
+#
+#Thu Mar 03 12:42:13 CST 2022
+Tests\ inheriting\ from\ AbstractTestBase\ should\ have\ name\ ending\ with\ ITCase=3ac3a1dc-681f-4213-9990-b7b3298a20bc
+ITCASE\ tests\ should\ use\ a\ MiniCluster\ resource\ or\ extension=f4d91193-72ba-4ce4-ad83-98f780dce581
```

Comment:
I think the problem right now is that we need to add a custom rule for tests extending `SourceTestSuiteBase` and white list them. Can you open a ticket for that? @JingGe

Reply:
I am a little bit short of the context, could your please describe the rule?

Reply:
If I understood correctly all test classes ending with `ITCase` need to have a `@RegisterExtension` with the minicluster or extend the `AbstractTestBase` but all classes extending the `SourceTestSuiteBase` currently fail the check because they do neither. I think we should extend the rule to also cover IT cases extending the `SourceTestSuiteBase`

Reply:
Tks, this helps a lot for the tests based on connector testing tools.